### PR TITLE
Improving documentation 

### PR DESCRIPTION
**Pull request by zoeyz101**
Fixes some spelling mistakes and added a couple of additional resources from the TensorFlow site for additional learning that may be needed by people starting out

### CODE REVIEW

**Comment by arjung** [comment]
@zoeyz101, can you revert the changes to .vscode/settings.json in this PR?

**Comment by zoeyz101** [comment]
> @zoeyz101, can you revert the changes to .vscode/settings.json in this PR?

Yeah I will remove that file. I think vscode just automatically adds it in for me.

File: g3doc/tutorials/adversarial_keras_cnn_mnist.ipynb [hunk 1]
```diff
@@ -77,19 +77,15 @@
...
         "\n",
         "In this tutorial, we illustrate the following procedure of applying adversarial\n",
         "learning to obtain robust models using the Neural Structured Learning framework:\n",
         "\n",
-        "1.  Creata a neural network as a base model. In this tutorial, the base model is\n",
+        "1.  Create a neural network as a base model. In this tutorial, the base model is\n",
         "    created with the `tf.keras` functional API; this procedure is compatible\n",
         "    with models created by `tf.keras` sequential and subclassing APIs as well.\n",
+        "    ([See more on keras here] (https://www.tensorflow.org/api_docs/python/tf/keras/Model))\n",
```
Comment (csferng) [inline g3doc/tutorials/adversarial_keras_cnn_mnist.ipynb:88]: Please remove the whitespace between `]` and `(`.

**Review by csferng — COMMENTED** [review]
Thanks! Looks good in general.

File: g3doc/tutorials/adversarial_keras_cnn_mnist.ipynb (hunk 1 above)
Comment (arjung) [inline g3doc/tutorials/adversarial_keras_cnn_mnist.ipynb:88]:
```suggestion
        "    For more information on Keras models in TensorFlow, see this [documentation](https://www.tensorflow.org/api_docs/python/tf/keras/Model).\n",
```

**Review by arjung — APPROVED** [review]

**Pull request merged**